bokeh/models/layouts.py

```diff
@@ -412,6 +412,10 @@ class Panel(Model):
     The child widget. If you need more children, use a layout widget, e.g. a ``Column``.
     """)
 
+    closable = Bool(False, help="""
+    Whether this panel is closeable or not. If True, an "x" button will appear.
+    """)
```

Comment:
We should be explicit here that the value of `tabs` will be mutated, and the panel removed